usr/src/man/man1m/zpool.1m

```diff
@@ -1484,12 +1484,12 @@ values.
 .It Fl T Sy u Ns | Ns Sy d
 Display a time stamp.
 Specify
-.Fl u
+.Sy u
```

Comment:
In FreeBSD `.Cm` macro is used here.
Not sure which is better...

Reply:
Perhaps `.Ar` would be even better.
I've seen examples like
```
.Fl k Ar kookfile
```

Reply:
I think .Li (literal) might be best here. While 'u' is an argument (.Ar), usually arguments are documented with a name, and it is expected that the user will replace them with a value for the argument, whereas in this case, the arguments are literally 'u' or 'd'

Reply:
My knowledge of manpages markup language is almost zero. Fixing obvious typos or inconsistensies is the maximum I can do now.

Yes, some better macros could probably be used. But wouldn't it better to make a general style-consistent rewrite/refactoring, and decouple of this one 4-line change ("do the same in 3 similar places")?

Reply:
@allanjude @avg-I, are you waiting for some more feedback from me?